<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbstractFunctionalityForm</class>
 <widget class="QWidget" name="AbstractFunctionalityForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>559</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../build-Framework-Desktop_Qt_5_7_0_MinGW_32bit-Debug/Images/lightning.jpg</normaloff>../../build-Framework-Desktop_Qt_5_7_0_MinGW_32bit-Debug/Images/lightning.jpg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <item>
      <widget class="QWidget" name="switchLabelBackground" native="true">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>31</height>
        </size>
       </property>
       <widget class="QLabel" name="labelSwitch">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>90</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../build-Framework-Desktop_Qt_5_7_0_MinGW_32bit-Debug/Images/Left.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="InteractiveButton" name="EditButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>0</y>
          <width>30</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../build-Framework-Desktop_Qt_5_5_1_MinGW_32bit-Debug/Images/Edit.png</normaloff>../../build-Framework-Desktop_Qt_5_5_1_MinGW_32bit-Debug/Images/Edit.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
       <widget class="InteractiveButton" name="EyeButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>30</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../build-Framework-Desktop_Qt_5_5_1_MinGW_32bit-Debug/Images/Eye.png</normaloff>../../build-Framework-Desktop_Qt_5_5_1_MinGW_32bit-Debug/Images/Eye.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="topSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="descriptionLayout">
     <item>
      <widget class="QLabel" name="descriptionLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>19</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="descriptionTextEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InteractiveButton</class>
   <extends>QPushButton</extends>
   <header>InteractiveButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
